builtin/providers/aws/resource_aws_vpc_dhcp_options.go

```diff
@@ -121,7 +121,7 @@ func resourceAwsVpcDhcpOptionsCreate(d *schema.ResourceData, meta interface{}) e
 	log.Printf("[DEBUG] Waiting for DHCP Options (%s) to become available", d.Id())
 	stateConf := &resource.StateChangeConf{
 		Pending: []string{"pending"},
-		Target:  "",
+		Target:  []string{},
```

Comment:
Thanks Clint, sorry this wasn't spotted during testing.

On 25 January 2016 at 20:43, Clint <notifications@github.com> wrote:

> In builtin/providers/aws/resource_aws_vpc_dhcp_options.go
> <https://github.com/hashicorp/terraform/pull/4767#discussion_r50752776>:
>
> > @@ -121,7 +121,7 @@ func resourceAwsVpcDhcpOptionsCreate(d *schema.ResourceData, meta interface{}) e
> >  	log.Printf("[DEBUG] Waiting for DHCP Options (%s) to become available", d.Id())
> >  	stateConf := &resource.StateChangeConf{
> >  		Pending: []string{"pending"},
> > -		Target:  "",
> > +		Target:  []string{},
>
> final follow up; I opened #4828
> <https://github.com/hashicorp/terraform/pull/4828> as a "fix". The bug
> only presents itself if []string{} is used in *creation*. It's normal for
> nil to be returned when we're deleting something, but DHCP Options here
> doesn't really provide a status.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hashicorp/terraform/pull/4767/files#r50752776>.
>